packages/info2man.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Info2man < Package
+  description 'Convert GNU info files to POD or man pages.'
+  homepage 'https://www.cskk.ezoshosting.com/cs/css/info2pod.html'
```

Comment:
I get a 404 for this url.

Reply:
Me too, that's the one used by Debian and the AUR. The next best URL is the salsa.debian.org one? @uberhacker let me know what you'd prefer me to use

Reply:
Good question.  The one I found that is least like a man or package page is http://freshmeat.sourceforge.net/projects/info2man.

Reply:
I'm having an issue with my HTTPS certificate on that site.

Reply:
That means a cert is not installed at that domain.